### PR TITLE
4.8.0 Release

### PR DESCRIPTION
**Pull request by jennifer-shehane**
- 4.8.0 Changelog
- Section for new spec file header #2852
- Update screenshots for error design changes #2853 
- [x] Test config options + isBrowser() changes #2697 
- [x] Update wording around 'test options' / 'test config'
- [x] `experimentalShadowDom` experimental flag #2858 

---

**Preview of main content for 'test config' feature**

Screen Shot 2020-06-08 at 2 16 00 PM

Screen Shot 2020-06-08 at 2 16 09 PM

### CODE REVIEW

File: source/_changelogs/4.8.0.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+**Misc:**
+
+- The design of errors and some iconography displayed in the Test Runner's Command Log have been updated. Addresses {% issue 7499 %}, {% issue 7542 %} and {% issue 7563 %}.
+- The commands in the Test Runner's Command Log now display in the same casing as the original command. Addresses {% issue 5773 %}.
+- The navigation links in the Test Runner now display the correct CSS styles when focused. Addresses {% issue 7491 %}.
+- {% url "`Cypress.config()`" config %} now has TypeScript types for the `taskTimeout` option. Addresses {% issue 7531 %}.
+- TypeScript types for {% url "`cy.screenshot()`" screenshot %} options and {% url "`Cypress.log()`" cypress-log %} have been updated to be more accurate. Addresses {% issue 7445 %}.
+- TypeScript types for `Cypress.automation` have been added. Addresses {% issue 7519 %}.
+- We not display a more accurate error message when passing a browser to the `--browser` flag that is not supported by Cypress. Addresses {% issue 6979 %}.
```
Comment (brian-mann) [inline source/_changelogs/4.8.0.md:34]: grammarrrrrrrgh

File: source/guides/guides/cross-browser-testing.md [hunk 2]
```diff
@@ -182,57 +182,33 @@ workflows:
...
-  }
-}
-
-const ignoreOn = (browser, fn) => {
-  if (!Cypress.isBrowser(browser)) {
-    fn()
-  }
-}
+// Run the test if Cypress is run via Firefox
+it('Download extension in Firefox', (), { browser: 'firefox' } => {
```
Comment (brian-mann) [inline source/guides/guides/cross-browser-testing.md:195]: Has erroneous additional parenthesis.

```
it('Download extension in Firefox', { browser: 'firefox' } => {
```

File: source/guides/references/configuration.md [hunk 3]
```diff
@@ -205,6 +205,40 @@ Cypress.config('pageLoadTimeout', 100000)
...
+  it('shows hamburger menu', () => {
+    cy.get('#header').find('i.menu').should('be.visible)
+  })
+})
+```
+
+### Single test configuration
+
+```js
+it('open product view', (), { waitForAnimations: false } => {
```
Comment (brian-mann) [inline source/guides/references/configuration.md:236]: more erroneous parenthesis

File: source/_changelogs/4.8.0.md (hunk 1 above)
Comment (jennifer-shehane) [reply source/_changelogs/4.8.0.md:34]: “We not display”, what’s wrong with that 😆

**Comment by CypressCecelia** [comment]
@jennifer-shehane  There's a typo in the Test Configuration documentation in the screenshot (couldn't find where the actual file was changed, I think that's a separate PR). In the "Note" with the red caution sign, the second sentence says 

> "The following...can be changed via per test configuration." 

It should be either via or per, not both.

**Comment by brian-mann** [comment]
> @jennifer-shehane There's a typo in the Test Configuration documentation in the screenshot (couldn't find where the actual file was changed, I think that's a separate PR). In the "Note" with the red caution sign, the second sentence says
> 
> > "The following...can be changed via per test configuration."
> 
> It should be either via or per, not both.

Maybe `with individual test configuration`?

**Comment by flotwig** [comment]
> couldn't find where the actual file was changed, I think that's a separate PR

same, can't find it, so let's go ahead with this PR as-is for now since it's not in this PR

**Pull request merged**